client/src/App.js

```diff
@@ -8,7 +8,7 @@ function App() {
   React.useEffect(() => {
     fetch("/showtracker")
       .then((res) => res.json())
-      .then((data) => setData(data[0]));
+      .then((data) => setData(data[2].artist));
```

Comment:
@Hanna-N9 I just put this in here as a placeholder to pull data from your showtracker db. We will remove it and add real data next sprint

Reply:
In my pull request, I decided to go back to the previous fetch (/api) instead of using showtracker for App.js. I had tested this.

Reply:
In my pull request, I decided to go back to the previous fetch (/api) instead of using showtracker for App.js. I had tested this.

Reply:
Sounds good I'll remove this and leave a comment on your PR!